packages/wonder-blocks-grid/components/row.js

```diff
@@ -88,33 +99,72 @@ export default class Row extends React.Component<{
         // that is still visible.
         const filteredContents = [];
         let hasVisibleCell = false;
+        let lastCell;
 
         for (const item of contents) {
             if (
-                !item.type ||
-                !item.props ||
-                !item.type.shouldDisplay ||
-                (typeof item.type.shouldDisplay === "function" &&
-                    item.type.shouldDisplay(item.props, gridSize))
+                typeof item === "object" &&
+                (!item.type ||
+                    !item.props ||
+                    !item.type.shouldDisplay ||
+                    (typeof item.type.shouldDisplay === "function" &&
+                        item.type.shouldDisplay(item.props, gridSize)))
             ) {
-                if (hasVisibleCell) {
+                // Figure out if we need to try and insert a gutter in-between
+                // cells. If the contents are flush then we don't insert any.
+                if (hasVisibleCell && !flush) {
```

Comment:
Does this mean that there won't be any spaces between the columns within this row?  I thought `flush` was all about the padding at the edges of the row.

Reply:
flush does both! See my comment above.